tests/testthat/test-distinct.R

```diff
@@ -102,9 +102,15 @@ test_that("distinct handles 0 columns edge case (#2954)", {
   expect_equal(nrow(distinct(tibble())), 0L)
 })
 
-test_that("distinct preserves order of the input variables (#3195)",{
+test_that("distinct respects order of the specified variables (#3195)",{
```

Comment:
```suggestion
test_that("distinct respects order of the specified variables (#3195, #6156)",{
```

Maybe worth referencing both issues